### ISSUE TITLE
getTeam() returns UnhandledPromiseRejectionWarning if team has 4 players roster

### ISSUE THREAD

**Issue by LaCTuK** [gigobyte/HLTV#260]
If team has 1 or more unassigned players (signed roster has 4 or less players) function **getTeam()** returns "UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): TypeError: Cannot read property 'split' of undefined" 
and can't get players at all (even if team has 4 known signed players).
Previous version (before #253 ) returned {name : "?", id : NaN} for unknown players if I am not mistaken.

It is interesting that your example in readme has no players (team SK, id 6137) and this example will not work.

Team ids for tests: **6137** (sk, 0 players in team), **7106** (eUnited, 4 players in team but still UnhandledPromiseRejectionWarning), **6238** (Ze Pug Godz, 4-players/1-"?"), **10304** (Triumph).

Long story short: 
HLTV.getTeam({id: 7106})
**Expected result**: return signed players only in players array
OR
return always 5 players but unassigned players will be like {name : "?", id: NaN / undefined}
**actual result**: UnhandledPromiseRejectionWarning

P.S. thx for your work

**Comment by lucasreppewelander** [comment]
this was fixed in #262 didn't know  of the #261 PR ¯\_(ツ)_/¯
